base_time_window/models/base_time_window.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import math
+from datetime import time
+
+from psycopg2.extensions import AsIs
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class TimeWindow(models.AbstractModel):
```

Comment:
@grindtildeath According to the naming conventions, 
filename should be: `time_window_mixin.py`
classname should be: `TimeWindowMixin`